### ISSUE TITLE
Jest error when running tests after upgrading to country-to-currency v2.x

### ISSUE THREAD

**Issue by jbarreiros** [thiagodp/country-to-currency#27]
Hi, thank you for maintaining this package.

With the latest version, `2.x`, our Jest tests started failing on a component using `country-to-currency`.

From what I can gather, it happens because Jest isn't ES-compatible yet, so it requires the CJS version of packages. This normally works fine, but for `country-to-currency`, Jest can't seem to figure out the location of the CJS package.

If I change `country-to-currency`'s `package.json` to the following, Jest runs as expected.

```
{
  "exports": {
    ".": {
      "import": "./index.esm.js",
      "require": "./index.cjs",
      "default": "./index.js"
    }
  }
}
```
  
Here is a minimal test case: https://stackblitz.com/edit/7trwxzru?file=README.md

Temporary workaround:
```
// jest.config.mjs
{
  moduleNameMapper: {
    '^country-to-currency$': '<rootDir>/node_modules/country-to-currency/index.cjs',
  }
}
```

Thank you.

**Comment by thiagodp** [comment]
Hello, @jbarreiros . Thank you for this Issue and your example. I'm publishing the fix in [v2.0.2](https://github.com/thiagodp/country-to-currency/releases/tag/v2.0.2)

**Comment by jbarreiros** [comment]
New `2.0.2` release works like a charm. Thank you.
